ui/lib/pki/addon/components/page/pki-configure-create.hbs

```diff
@@ -91,13 +91,9 @@
     data-test-configuration-empty-state
   />
   <div class="field is-grouped box is-fullwidth is-bottomless">
-    <div class="control">
-      <button type="button" class="button is-primary" disabled={{true}} data-test-pki-config-save>
-        Done
-      </button>
-      <button type="button" class="button has-left-margin-s" {{on "click" @onCancel}} data-test-pki-config-cancel>
-        Cancel
-      </button>
-    </div>
+    <Hds::ButtonSet>
```

Comment:
This is your call - but where I've added Hds::Button set I've replaced the extra surrounding div with the css above with `<hr class="has-background-gray-100" />`